patches/third_party-blink-renderer-core-frame-navigator.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/frame/navigator.cc b/third_party/blink/renderer/core/frame/navigator.cc
+index 6d0c3d3961d6ef56f9780eea101d64f36fde0637..906a10149fd2ddb1c72e5692742682522eb26ebe 100644
+--- a/third_party/blink/renderer/core/frame/navigator.cc
++++ b/third_party/blink/renderer/core/frame/navigator.cc
+@@ -113,6 +113,7 @@ void Navigator::Trace(blink::Visitor* visitor) {
```

Comment:
having some trouble patching this in a more clean fashion; since `Trace` is matching the below calls too ☹️

Reply:
what is the reason for this patch?

Reply:
the solution you proposed is a better way to do it 😄 But to answer this question, the reason was so that `brave_` would have a trace call associated with it (required; otherwise throws compile error)